src/differentiation/compute_jacobian_ad.jl

```diff
@@ -316,3 +333,17 @@ function forwarddiff_color_jacobian!(J::AbstractMatrix{<:Number},
     end
     return J
 end
+
+# fast version of FiniteDiff._colorediteration! for the case where J and sparsity have the same sparsity pattern
+@inline function _colorediteration!(Jsparsity::SparseMatrixCSC,vfx,colorvec,color_i,ncols)

```

Comment:
Shouldn't this just go to FiniteDiff.jl?

Reply:
PR JuliaDiff/FiniteDiff.jl#124